### ISSUE TITLE
[Bug]: Failed to download php parser on windows

### ISSUE THREAD

**Issue by LaurinStapf** [laravel/vs-code-extension#92]
### Extension Version

v0.1.12

### PHP Binary

Herd

### Operating System

Windows

### What happened?

The extension can't download the latest version of php parser because the name in the extension is wrong.
It says the following:

> Failed to download binary for Laravel extension: php-parser-v0.1.17-x64-win32

This is probably due to the fact that the latest version is named as follows: `php-parser-v0.1.31-x64-win32.exe`.

### Mimimal Code Sample

```
File Downloader Log:

Starting download from https://github.com/laravel/vs-code-php-parser-cli/releases/download/v0.1.17/php-parser-v0.1.17-x64-win32
ERROR: Request failed with status code 404. Technical details: {"message":"Request failed with status code 404","name":"AxiosError","stack":"AxiosError: Request failed with status code 404\n
```

**Comment by MasterZydra** [comment]
Related with #86

**Comment by joetannenbaum** [comment]
We just released v0.1.13 that should have better Windows support, can you verify that this is still an issue after upgrading?

**Comment by LaurinStapf** [comment]
The download of the php-parser is now working, but there is another error:

```
Error:
 App Bindings

Could not open input file: {code}

php "{code}" c:\Users\StapfL\Herd\ess\vendor\_laravel_ide\discover-d4f39ea8c8afd10ce595be32bd60d7a7.php
```

**Comment by joetannenbaum** [comment]
Good catch, I'll release the fix in the next round but in the short term can you try and set the PHP Command setting to simply `php` and see if that works?

**Comment by isabella-projects** [comment]
If my project structure is different, e.g at root is **src**, .gitignore, README.md etc.. where inside the **src** is the actual project and the vendor folder, the extension does not start. Can we have fix for that as well? It would be nice to auto detect the vendor folder if it is not at the root level.

**Comment by LaurinStapf** [comment]
> Good catch, I'll release the fix in the next round but in the short term can you try and set the PHP Command setting to simply `php` and see if that works?

It works, more or less...
Just to be sure, should things like eloquent field completion and method completion work fine?

And I have discovered a bug, the translation auto-completion do not work with subfolders (but this could also be cause this is still beta ;-) )

**Comment by hussain4real** [comment]
it does'nt download the binary and got this error:
`[error] /bin/sh: /Users/amisha/Library/Application Support/Code: is a directory`

using M1

**Comment by joetannenbaum** [comment]
Binary is compiled for all supported platforms now, if you encounter other errors please open a new issue.
